src/codeu/chat/DerbyStore.java

```diff
@@ -103,15 +147,17 @@ public boolean checkUsernameExists(String name) throws SQLException {
 		return user == null;
```

Comment:
Won't this return true if the username does **not** exist? Some unit tests for these functions would be awesome.